integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -599,11 +599,6 @@ case class LoadTable(
         // Once the data load is successful delete the unwanted partition files
         try {
 
-          // shutdown dictionary server thread
-          if (carbonLoadModel.getUseOnePass) {
```

Comment:
If this is not shut down, the next load operate will still try to start another dictionary server. 
Do you want to reuse this dictionary server in next load?